lib/puppet/type/quagga_static_route.rb

```diff
@@ -0,0 +1,66 @@
+Puppet::Type.newtype(:quagga_static_route) do
+  @doc = %q{
+    This type provides the capability to manage static routes within puppet.
+
+      Example:
+
+        quagga_static_route {'172.16.2.0/23':
+          ensure      => present,
+          gateway     => '192.168.1.10',
```

Comment:
For subnet mask or slash notation - would you suggest making that a part of the `name` parameter (it is now since I only opted for slash notation)?